### PR TITLE
Implement Post Creation Success Alert View

### PR DESCRIPTION
**Pull request by OnyekachiSamuel**
**What this PR does**:
This PR handles the implementation of Post Creation Alert View
### Checklist

- [x] Compiler warnings resolved
- [x] Linter warnings resolved
- [x] User-facing strings in Localizable.strings file
- [x] Unused code, print statements, and comments removed

**Which issue(s) this PR fixes**:
#64 

**Special notes for reviewers**:
To test this feature out, call PostCreationSuccessAlert.shared.showAlert() from maybe a button tap from any view controller.

Simulator Screen Shot - iPhone 11 Pro - 2020-05-30 at 03 34 30

### CODE REVIEW

File: FightPandemics/Features/Post/CreatePost/PostCreationSuccessAlert.swift [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    private func setupViews() {
+        backgroundColor = #colorLiteral(red: 0, green: 0, blue: 0, alpha: 0.2727686216)
+        containerView.backgroundColor = .white
+        containerView.layer.cornerRadius = 10
+        containerView.layer.masksToBounds = true
+
+        imageView.contentMode = .scaleAspectFill
+
+        titleLabel.textColor = #colorLiteral(red: 0.1568627451, green: 0.1568627451, blue: 0.1568627451, alpha: 1)
```
Comment (ethanswift) [inline FightPandemics/Features/Post/CreatePost/PostCreationSuccessAlert.swift:59]: All colors are available in the palette, are they different or you just wanted to use these colors? (they seems to be more precise though, I'm just asking)

**Review by ethanswift — CHANGES_REQUESTED** [review]
Everything is OK and your code is perfect as always, just take a look at the colors and see if you want to use the palette colors or not

**Review by stavares843 — APPROVED** [review]

**Review by ethanswift — APPROVED** [review]

File: FightPandemics/Features/Post/CreatePost/PostCreationSuccessAlert.swift (hunk 1 above)
Comment (OnyekachiSamuel) [reply FightPandemics/Features/Post/CreatePost/PostCreationSuccessAlert.swift:59]: Thanks Ethan, I will check that out and update accordingly.

Comment (OnyekachiSamuel) [reply FightPandemics/Features/Post/CreatePost/PostCreationSuccessAlert.swift:59]: I will be maintaining this for now, just like you said, this one is more precise and accurate, the quality is lost when I tried using the color functions we have already.

**Pull request merged**